src/components/Drawer/Drawer.tsx

```diff
@@ -93,6 +93,7 @@ export const Drawer: React.FC<Props> = ({
                     {...block({ [getStatus()]: state !== 'entering' })}
                     aria-hidden={!isShown}
                     {...rest}
+                    role="group"
```

Comment:
I've put "group" role here is it okay or should we use other one ?

Reply:
Good question. Looking at the docs I think it is a live-region and us such should have one of the associated roles. see https://www.w3.org/TR/wai-aria-1.2/#ref-for-dfn-live-region-1.

Reply:
hmm, really a good question :) It looks like for Source we can use role"status" with aria-live="polite". On another hand, the Drawer component from the OneUI perspective is a collapsible element that can wrap any data from the application, so I think the application should decide which role is most suitable. As far as I know, a reader reads all text inside the aria-live block when the element appeared. In this scenario, all elements will be read, but we need only read the last added items, thus we need to add aria-live only to the items (application level). 
My conclusion:

- use role="group" for the Drawer component. The application is responsible for the accessibility inside the Drawer. 
- use "aria-live" for the Drawer, but in this case, we need to dive into details how to use it, how to exclude some text, 
how to use us earia-atomic, aria-relevant etc.

I'd prefer the first point.